mongodbforms/fieldgenerator.py

```diff
@@ -94,6 +94,17 @@ def generate(self, field, **kwargs):
         raise NotImplementedError('%s is not supported by MongoForm' %
                                   field.__class__.__name__)
 
+    def get_choices(self, field, widget=None):
```

Comment:
I added another method rather than inlining in `get_field_choices` incase a user expects the signature of `get_field_choices` to remain as is. If you think it is unlikely I can update the original method.